src/app/pages/contact/embedded.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Form from './form';
+import './embedded.scss';
+
+// File is js because it is dynamically imported (and minimal)
+export default function EmbeddedForm() {
+    const [submitted, setSubmitted] = React.useState(false);
+
+    if (submitted) {
+        return <ThankYou />;
+    }
+
+    return (
+        <main id="maincontent" className="embedded-contact">
+            <h1>Report an issue</h1>
+            <div>
+                Need to suggest a content correction instead? Fill out the{' '}
+                <a href="/errata" target="_blank">
```

Comment:
This is outside the scope of these changes but just wondering if we have a card to track this clipping issue on the prod errata page:
![image](https://github.com/user-attachments/assets/82b910af-23ac-45b5-bc43-32f2cf4286af)

Reply:
This is the errata page with no book in the URL. With a book, the header gets populated and the selectors at the top are pushed down. Without a book, the page is useless.
https://openstax.org/errata/?book=Principles%20of%20Accounting%2C%20Volume%201%3A%20Financial%20Accounting

Reply:
Do we have the book info inside this form to make the right url? Otherwise it seems like we should remove the link.

Reply:
D'oh!